torch/csrc/jit/script/parse_string_literal.h

```diff
@@ -67,11 +67,14 @@ inline std::string parseStringLiteral(
       case 't':
         c = '\t';
         break;
-      case 'h':
+      case 'x':
```

Comment:
in both C++ and Python, the hex starts with 'x', not 'h'... So I changed it to x.